src/__experimental__/components/decimal/decimal.component.js

```diff
@@ -54,6 +56,10 @@ class Decimal extends React.Component {
     }
   }
 
+  startWiggleAnimation = () => this.setState({ isWiggling: true })
```

Comment:
According to the comment above. What about just `startAnimation` and `stopAnimation`